From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@AnuragMehrotra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (anuragm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
add language support

Commit:
Update FAQ

v-geberr committed:
bing spell check no longer free

v-geberr committed:
note about hitcount accuracy in azure portal

Commit:
Merge pull request #29318 from MicrosoftDocs/Banani-Rath-patch-1

Update .openpublishing.publish.config.json

v-geberr committed:
transfer of ownership

Commit:
zh-CN entry in table

PRMerger19 committed:
Merge pull request #29221 from v-geberr/1204-1238-example-utterances

corrected entity name

PRMerger14 committed:
Merge pull request #29252 from v-geberr/1204-train-test

1204 train test

v-geberr committed:
minor correction

PRMerger19 committed:
Merge pull request #29193 from v-geberr/1204-0928-patterns-support

deprecated feature

Commit:
Merge pull request #29245 from v-geberr/1204-publish

minor fixes

v-alje committed:
Resolving merge conflict with master

PRMerger13 committed:
Merge pull request #29230 from v-geberr/124-0110-add-entities

corrected h2 name

v-alje committed:
Merge branch 'release-luis-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into release-luis-ga

PRMerger10 committed:
Merge pull request #29328 from v-geberr/1205-collaborator-howto

transfer of ownership

v-geberr committed:
requested change

Ja-Dunn committed:
Merge pull request #29210 from v-geberr/1204-qs-new-app

switching out images

v-alje committed:
Merge branch 'release-luis-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into release-luis-ga

v-alje committed:
Merge branch 'release-luis-ga' of https://github.com/MicrosoftDocs/azure-docs-pr into release-luis-ga

v-geberr committed:
fixing note syntax

Commit:
Merge pull request #29243 from v-geberr/1204-manage-keys

-critical - new endpoint hit quota values

PRMerger12 committed:
Merge pull request #29247 from v-geberr/1204-active-learning

change title to reflect new name in UX

Commit:
Update luis-get-started-cs-get-intent.md

Commit:
Update luis-get-started-java-get-intent.md

Commit:
Update luis-get-started-js-get-intent.md

Commit:
Update luis-get-started-node-get-intent.md

Commit:
Update luis-get-started-php-get-intent.md

Commit:
Update luis-get-started-python-get-intent.md

v-geberr committed:
fixing syntax based on warning

Ja-Dunn committed:
Merge pull request #29203 from v-geberr/1204-1116-supportedlanguages

french datetimev2 now supported via Carol

PRMerger15 committed:
Merge pull request #29244 from v-geberr/1204-dashboard

minor changes

Commit:
Merge pull request #29291 from Kaiqb/luis-qs-fix

Update luis-get-started-cs-get-intent.md

Commit:
Update luis-get-started-ruby-get-intent.md

Commit:
table update

Commit:
Update luis-quickstart-cs-add-utterance.md

Commit:
Update luis-quickstart-java-add-utterance.md

Commit:
Update luis-quickstart-node-add-utterance.md

Commit:
Update luis-quickstart-php-add-utterance.md

Commit:
Update luis-quickstart-python-add-utterance.md

Commit:
Merge pull request #29260 from DeniseMak/12-4-chinese-domains

language support column

PRMerger12 committed:
Merge pull request #29238 from DeniseMak/12-4-entity-deprecation

table update for deprecated entities, date, Language Understanding

Commit:
Update luis-quickstart-ruby-add-utterance.md

v-geberr committed:
small changes to be more concise

PRMerger16 committed:
Merge pull request #29345 from Kaiqb/luis-qs-fix

Luis qs fix

PRMerger15 committed:
Merge pull request #29347 from v-geberr/1205-dashboard

small changes to be more concise

v-geberr committed:
clarify opening sentence

v-alje committed:
Bringing even with master.

Commit:
Update luis-get-started-cs-get-intent.md

Commit:
Update luis-get-started-java-get-intent.md

Commit:
Update luis-get-started-js-get-intent.md

Commit:
Update luis-get-started-node-get-intent.md

Commit:
Update luis-get-started-php-get-intent.md

Commit:
Update luis-get-started-python-get-intent.md

Commit:
Update luis-get-started-ruby-get-intent.md

Commit:
Update luis-quickstart-cs-add-utterance.md

Commit:
updates to logging, date

Commit:
fix typo in publish step

Commit:
Merge pull request #29354 from Kaiqb/luis-link-fix

Luis link fix

v-geberr committed:
simplier term

Commit:
example intent numbers, spell check edit

v-geberr committed:
fixing wording to be more clear

Lisaco88 committed:
Merge pull request #29361 from DeniseMak/12-5-tutorial-fix

Tutorial: fix typo in publish step

v-geberr committed:
change pub date

Commit:
Fix typo

PRMerger17 committed:
Merge pull request #29365 from v-geberr/1205-howto

1205 howto

PRMerger10 committed:
Merge pull request #29371 from v-geberr/release-luis-ga

change pub date

PRMerger14 committed:
Merge pull request #29262 from DeniseMak/12-4-update

Update FAQ

v-geberr committed:
reworked to be more scannable

v-geberr committed:
rework symbol for deprecated and coming soon

v-geberr committed:
fixed tokens based on Carol feedback

v-geberr committed:
fixed korean based on Carol feedback

v-geberr committed:
cleanup

v-geberr committed:
fix

v-geberr committed:
changed pub date for GA

PRMerger17 committed:
Merge pull request #29399 from v-geberr/1206-change-pub-date

changed pub date for GA

v-shils committed:
Merge pull request #29390 from v-geberr/1206-supported-languages-2

reworked to be more scannable

v-geberr committed:
fixed story 17728

v-geberr committed:
fix

PRMerger17 committed:
Merge pull request #29415 from v-geberr/1206-version-story-17728

1206 version story 17728

v-alje committed:
Bringing even with master.

Commit:
link to reference

Commit:
Merge pull request #29392 from v-geberr/1206-prebuilt-entity-reference

rework symbol for deprecated and coming soon

PRMerger18 committed:
Merge pull request #29419 from DeniseMak/12-6-domain-link

GA: add link to prebuilt domains reference for Chinese notes

v-geberr committed:
new limit for authoring api

PRMerger20 committed:
Merge pull request #29430 from v-geberr/1206-manage-keys-1M-auth

new limit for authoring api

Commit:
Sentence clarifying source of endpoint hits.

PRMerger16 committed:
Merge pull request #29435 from DeniseMak/12-6-dashboard

Sentence clarifying source of endpoint hits.

Commit:
updates to spell check

v-alje committed:
Bringing even with master.

Commit:
Merge pull request #29461 from DeniseMak/12-4-update

updates to spell check

Commit:
clarify where spelling corrections are found in JSON

PRMerger16 committed:
Merge pull request #29465 from DeniseMak/12-6-update-faq-spell-ux

clarify where spelling corrections are found in JSON

Commit:
fixes to OnDevice

Commit:
Fixes to Places

Commit:
Places entities

PRMerger20 committed:
Merge pull request #29468 from DeniseMak/12-6-prebuilt-domains-fix

fixes to OnDevice

Commit:
Fix to Places intents

PRMerger14 committed:
Merge pull request #29484 from DeniseMak/12-6-prebuilt-domains-fix

Fix to Places intents

Commit:
Fixes to OnDevice intents

Commit:
Merge pull request #29488 from DeniseMak/12-7-prebuilt-domains-fix

Fixes to OnDevice intents

Commit:
edits to troubleshooting hit count and owner transfer, date in metadata

PRMerger19 committed:
Merge pull request #29490 from DeniseMak/12-7-faq

edits to troubleshooting hit count and owner transfer, date in metadata

Commit:
edit to deleting data

Commit:
update dates

PRMerger16 committed:
Merge pull request #29496 from DeniseMak/12-7-faq3

edit to deleting data

Commit:
Update index.yml

Commit:
update programmatic editing question

Commit:
Add step for new parameter

v-alje committed:
Bringing even with master.

GitHubber17 committed:
Merge pull request #29504 from Kaiqb/luis-lp-link

Update index.yml

Commit:
fix link

Commit:
minor edit

PRMerger19 committed:
Merge pull request #29506 from DeniseMak/12-7-faq4

Add spellcheck step for new parameter

v-alje committed:
Resolving merge conflict

v-geberr committed:
luis.ai to www.luis.ai

PRMerger10 committed:
Merge pull request #29660 from v-geberr/1208-fix-broken-links

luis.ai to www.luis.ai

v-geberr committed:
removing text that doesn't go with new image

v-geberr committed:
changed to c# as next step

PRMerger15 committed:
Merge pull request #29662 from v-geberr/1208-create-app

removing text that doesn't go with new image

v-geberr committed:
change term, and cleanup

v-alje committed:
Resolving merge conflict

PRMerger20 committed:
Merge pull request #29668 from v-geberr/1208-supported-languages

change term, and cleanup

cwatson-cat committed:
Upd monitoring topics to remove references to classic portal.

v-alje committed:
Bringing even with master.

HeidiSteen committed:
explain the .50 score

v-alje committed:
Bringing even with master.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

v-viinde committed:
Updated

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
Revised the sentiment analysis edit, plus changed the H2 handle results to view results

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
edits

HeidiSteen committed:
Removed a paragraph break that I introduced unnecessarily

v-alje committed:
Bringing even with master.

barbkess committed:
new loading guidance and overview

barbkess committed:
fixes

barbkess committed:
fix

v-alje committed:
Bringing even with master.

billmath committed:
fixing copy paste error

v-viinde committed:
USER STORY 1144818 SaaS App Tutorial: ArcGIS Online Update

v-viinde committed:
Merge branch 'master' into arcgisonlineupdate

ruthogunnaike committed:
Ruby Quickstart

dubansal committed:
Removed extra \ characters

CarlRabeler committed:
ha

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
edit for clarity

JeffGoldner committed:
Add new marketplace items

MikeDodaro committed:
Standardizing TOCs

tfitzmac committed:
added resources section

CarlRabeler committed:
ha2

MikeDodaro committed:
move Tutorials section

MikeDodaro committed:
Use and Display Requirements -> How-Tos

anavinahar committed:
Update IP on NSG

@malopMSFT this came up as part of a ww communities email and also checked in NRP. FYI.

ggailey777 committed:
Updated links to our docs.

MikeDodaro committed:
Standardize Custom Search Quickstarts.

billmath committed:
fixing release notes

billmath committed:
updating

PRMerger20 committed:
Merge pull request #29887 from dubansal/patch-1

Removed extra \ characters

muralikk committed:
Added Germany as supported regions

Commit:
task 343: change User Variable text to internal use only

PRMerger10 committed:
Merge pull request #29924 from ggailey777/patch-5

Updated links to our docs.

GitHubber17 committed:
Merge pull request #29913 from JeffGoldner/master

Add new marketplace items

Commit:
author updates

Commit:
updates to raise acrolinx

SaloniSonpal committed:
Remove note about old dns for verify-full

PRMerger13 committed:
Merge pull request #29928 from muralikk/patch-16

Added Germany as supported regions

v-shils committed:
Merge pull request #29919 from tfitzmac/1212resources

added resources section

iainfoulds committed:
Merge branch 'master' of https://github.com/iainfoulds/azure-docs-pr

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-ta

HeidiSteen committed:
LCC edits

Commit:
task 343: put HTTP headers in code format

Commit:
Merge pull request #29765 from HeidiSteen/heidist-ta

explain the .50 score

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

PRMerger17 committed:
Merge pull request #29922 from anavinahar/patch-30

Update IP on NSG

PRMerger18 committed:
Merge pull request #29929 from georgewallace/autoauthorupdates

automation : author updates 1

Commit:
updates to authors

ChiragPavecha committed:
Update cloud-services-certs-create.md

Commit:
Merge pull request #29931 from dksimpson/dean-343

Dean 343

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into load

Commit:
acrolinx feedback

v-shils committed:
Merge pull request #29912 from MikeDodaro/standardizeTOCs

Standardizing TOCs

Commit:
updates for acrolinx

CarlRabeler committed:
typo

ChiragPavecha committed:
Update cloud-services-certs-create.md

Added -KeyLength 2048 -KeySpec "KeyExchange" to generate correct certificate with private key

Commit:
getting acrolinx above 80

Commit:
Merge pull request #29683 from cwatson-cat/12-9-ClassicClnUp-Monitoring

Upd monitoring topics to remove references to classic portal.

Commit:
over 80 yet?

billmath committed:
updating

barbkess committed:
guidance updates

PRMerger17 committed:
Merge pull request #29932 from georgewallace/autoauthorupdates

updates to authors 2

mumian committed:
add blog link

Commit:
updates 3

PRMerger16 committed:
Merge pull request #29933 from ChiragPavecha/patch-1

Update cloud-services-certs-create.md

rwike77 committed:
Adding entry.

v-shils committed:
Merge pull request #29911 from CarlRabeler/ha

Ha

Commit:
Merge pull request #29936 from georgewallace/autoauthorupdates

automation author updates 3

Commit:
author updates 4

PRMerger17 committed:
Merge pull request #29938 from mumian/1213-seeblog

add blog link

MikeDodaro committed:
Revise style of endpoint summary docs

Commit:
Merge pull request #29941 from georgewallace/autoauthorupdates

automation: author updates 4

Commit:
automation author updates 5

Commit:
acrolinx ypdates

nitinme committed:
Updates to HDI overview

GitHubber17 committed:
Merge pull request #28426 from v-viinde/sumtotalcentral

USER STORY 1087732 SaaS App Tutorial: SumTotalCentral

PRMerger18 committed:
Merge pull request #29930 from SaloniSonpal/master

Remove note about old dns for verify-full

iainfoulds committed:
Freshness updates and revisions

PRMerger12 committed:
Merge pull request #29943 from georgewallace/autoauthorupdates

automation author updates 5

MikeDodaro committed:
Boldface endpoint lables

Commit:
updates to author

rnagpal committed:
Changing the min throughput to 1000

Changing 1100 RU/s to 1000 RU/s as the minimum throughput.

@mimig1 : FYI

v-shils committed:
Merge pull request #29886 from v-viinde/arcgisonlineupdate

USER STORY 1144818 SaaS App Tutorial: ArcGIS Online Update

Commit:
corrections for acrolinx

rnagpal committed:
Removed preview tag from Data Explorer

nitinme committed:
validation fixes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-updates

YuriDio committed:
updating article

Commit:
1144341 - Update Advisor index.md with link to CLI documentation & changed REST to REST API

Commit:
updates for acrolinx

jeffpatt24 committed:
Update storage-sync-files-planning.md

Addeed East US to region availability list

PRMerger13 committed:
Merge pull request #29937 from rwike77/patch-3

Adding entry.

Commit:
updates for acrolinx

jeffpatt24 committed:
Update storage-files-faq.md

Added East US to region availability list

PRMerger17 committed:
Merge pull request #29953 from YuriDio/UpdatesDez10

updating article

rmca14 committed:
Merge pull request #29847 from barbkess/load

new loading guidance and overview

PRMerger18 committed:
Merge pull request #29951 from rnagpal/patch-26

Changing the min throughput to 1000

v-alje committed:
Merge pull request #29835 from MicrosoftDocs/release-luis-ga

Release luis ga

rmca14 committed:
Merge pull request #29704 from ruthogunnnaike/RubyQS

Ruby Blob Quickstart

cwatson-cat committed:
Updated sendgrid include file to replace reference to classic portal.

Lisaco88 committed:
Merge pull request #29942 from MikeDodaro/reviseEndpoints

Revise style of endpoint summary docs

cwatson-cat committed:
Removed reference to classic portal in Security overview topic

rmca14 committed:
Merge pull request #29947 from nitinme/nitinme-hdi-updates

Updates to HDI overview

PRMerger15 committed:
Merge pull request #29902 from billmath/securityupdate

fixing copy paste error

Commit:
Merge pull request #29959 from cwatson-cat/12-13-ClassicUpd-SendGrid

Updated sendgrid include file to replace reference to classic portal.

MicrosoftGuyJFlo committed:
Add note for authentication data on change from group to everyone

Lisaco88 committed:
Merge pull request #29948 from iainfoulds/vmfreshness121317-batch1

VM freshness updates

PRMerger17 committed:
Merge pull request #29961 from cwatson-cat/12-13-ClassicClnUp-SecurityOV

Removed reference to classic portal in Security overview topic

Lisaco88 committed:
Merge pull request #29950 from georgewallace/autoauthorupdates

automation : updates to author 6

PRMerger20 committed:
Merge pull request #29956 from jeffpatt24/patch-8

Patch 8

MicrosoftGuyJFlo committed:
Add note for authentication data on change from group to everyone

cwatson-cat committed:
Removed minor references to Classic portal in 3 Storage topics.

v-alje committed:
Resolving merge conflict

Commit:
Merge pull request #29966 from cwatson-cat/12-13-ClassicClnUp-Storage

Removed minor references to Classic portal in 3 Storage topics.

curtand committed:
re-added Azure Information Protection after regression

rmca14 committed:
Merge pull request #29954 from cmcclister/1144341

1144341 - Landing page updated due to TOC update

PRMerger18 committed:
Merge pull request #29964 from MicrosoftGuyJFlo/SSPRUpdateFromPM

Add note for authentication data on change from group to everyone

v-alje committed:
Merge pull request #29967 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger19 committed:
Merge pull request #29968 from curtand/aip1213

re-added Azure Information Protection after regression
